### PR TITLE
Meenu rebranding live price and trading spec

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trading-specification/_use-trading-spec-columns.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import { Localize } from 'components/localization'
+
+const liveMarketColumnHelper = createColumnHelper<THeaders>()
+
+const useLiveColumns = () => {
+    const columns = useMemo(() => {
+        return [
+            liveMarketColumnHelper.accessor('instrument', {
+                header: () => (
+                    <TableHeaderCell text={<Localize translate_text="Instrument name" />} />
```
Comment (habib-deriv) [inline src/pages/trading-specification/_use-trading-spec-columns.tsx:14]: For all the Localization let's use _t_STRING_t_

File: src/pages/trading-specification/data/_commodities_specification.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        contract_size: 5000,
+        base_currency: 'XAG',
+        minimum_size: 0.01,
+        minimum_spread: 0.013,
+        target_spread: 0.02,
+        max_effective_leverage: 1000,
+        margin_requirement: 0.1,
+        swap_long: 0,
+        swap_short: 0,
+        trading_hours: 'Sun 22:05 - Friday 21:55 GMT Daily Break: 22:00-22:05',
```
Comment (habib-deriv) [inline src/pages/trading-specification/data/_commodities_specification.ts:15]:
```suggestion
        trading_hours: 'Sun 22:05 - Friday 21:55 GMT Daily Break: 22:00-22:05',
        eu: true;
```

File: src/pages/trading-specification/data/_crypto_specification.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        contract_size: 1,
+        base_currency: 'AUD',
+        minimum_size: 0.01,
+        minimum_spread: 18.15,
+        target_spread: 20.905,
+        max_effective_leverage: 100,
+        margin_requirement: 1,
+        swap_long: -0.06,
+        swap_short: -0.06,
+        trading_hours: 'Sun 22:05 - Friday 21:55 GMT Daily Break: 22:00-22:05',
```
Comment (habib-deriv) [inline src/pages/trading-specification/data/_crypto_specification.ts:15]:
```suggestion
        trading_hours: 'Sun 22:05 - Friday 21:55 GMT Daily Break: 22:00-22:05',
         eu: true;
```

File: src/pages/trading-specification/_use-trading-spec-columns.tsx (hunk 1 above)
Comment (NikitK-deriv) [reply src/pages/trading-specification/_use-trading-spec-columns.tsx:14]: We have information. regarding this in wiki: https://wikijs.deriv.cloud/en/Frontend/deriv-static/07-translation-workflow/Translation-Key-Extraction-POC#best-practices

**Pull request merged**